translation.json

```
@@ -0,0 +1,14 @@
+{
+  "widgets": {
+    "chart": {
+      "column": {
+        "title": "Column Chart",
+        "instruction": "Start configuring your component and the chart will appear here.",
+        "total": "<i class=\"fa fa-tachometer\"></i> Total<br><span class=\"total\">&mdash;</span>",
+        "updatedAt": "<i class=\"fa fa-clock-o\"></i> Last updated<br><span class=\"updatedAt\">&mdash;</span>",
+        "category": "Category",
+        "count": "Count of"
```

The column name should be part of the template because "Count of X" could translated with "X" in the beginning for some languages.

The same feedback should be applied to all other chart types that support summary mode.

```suggestion
        "count": "Count of {{column}}"
```